packages/cli/src/lib/bundler/config.ts

```diff
@@ -232,12 +233,50 @@ export async function createConfig(
     require.resolve('react-refresh'),
   ];
 
+  const mode = isDev ? 'development' : 'production';
+  const optimization = optimizationConfig(options);
+
+  if (
+    process.env.EXPERIMENTAL_MODULE_FEDERATION === 'true' &&
```

Comment:
Only check if it's truthy, it's common to expect for example `EXPERIMENTAL_MODULE_FEDERATION=1` to work. Same goes for the other one.

If you really want to be able to explicitly disable, then we could use `yn`, but you'd need to update other usages too.

Reply:
:+1: df694e957451